### PR TITLE
CO-2607 Set paiement term as imediate for web donation

### PR DESCRIPTION
**Pull request by eicher31**

### CODE REVIEW

**Comment by eicher31** [comment]
@ecino
Is the error due to my changes?

File: website_event_compassion/forms/donation_form.py [hunk 1]
```diff
@@ -128,6 +128,9 @@ def form_before_create_or_update(self, values, extra_values):
             })
 
         def _form_create(self, values):
+            # Set payment term as immediate
+            values['payment_term_id'] = self.env.ref(
```
Comment (ecino) [inline website_event_compassion/forms/donation_form.py]: Why not including at line 127 where we already define all invoice values?

**Review by ecino — CHANGES_REQUESTED** [review]

**Comment by ecino** [comment]
> @ecino
> Is the error due to my changes?

No it was another error that should be fixed now.

File: website_event_compassion/forms/donation_form.py (hunk 1 above)
Comment (ecino) [reply website_event_compassion/forms/donation_form.py]: Could you check it?

Comment (eicher31) [reply website_event_compassion/forms/donation_form.py]: Yes it's done

**Review by ecino — APPROVED** [review]

**Pull request merged**